docs/apps/class_based_skeleton.rst

```diff
@@ -0,0 +1,12 @@
+.. _class_skeleton_app:
+
+############
+Skeleton app
+############
+
+This is an `example application`_. It shows basics of initializing your application in a class based fashion, some conventions you need to follow and basics of working with UI elements.
+
+.. _example application: https://github.com/CRImier/pyLCI/tree/master/apps/example_apps/class_based_skeleton
```

Comment:
ZeroPhone/ZPUI instead of CRImier/pyLCI , sorry, forgot that link in the skeleton app docs page! I'll remove pyLCI mentions from the docs sometime soon, but please change this before merging =)